support_branding/README.rst

```diff
@@ -69,6 +85,7 @@ Contributors
 
 * Holger Brunn <hbrunn@therp.nl>
 * Stefan Rijnhart <srijnhart@therp.nl>
+* Angel Moya <odoo@tecnativa.com>
```

Comment:
after you rewrote the js part to v9 api, I'd consider it a serious contribution. As it is, I think this is not warranted